projects/ad9083_evb/common/ad9083_evb_bd.tcl

```diff
@@ -123,7 +123,7 @@ ad_ip_parameter dma_clk_generator CONFIG.PRIM_SOURCE No_buffer
 ad_connect $sys_cpu_clk dma_clk_generator/clk_in1
 ad_connect $sys_cpu_resetn dma_clk_generator/resetn
 
-ad_disconnect sys_250m_clk sys_ps8/pl_clk1
+ad_disconnect $sys_dma_clk $sys_dma_clk_source
```

Comment:
I think this hardcoding should be removed, but I'm not sure if this is the best way. Maybe defining a $sys_custom_dma_clk variable would be better and use that to drive the DMA path?